### ISSUE TITLE
The App Can't actually read Image URLs

### ISSUE THREAD

**Issue by jswope00** [jswope00/AI-MicroApps#4]
Turns out, telling the AI "Please write alt text for the following image: [List of Image URLs]" does not work. GPT models, at least, won't access those URLs and view the image. Sometimes they *pretend* that they can by reading the URL of the image, but they actually aren't. 

To fix this, we'll need to find some way to extricate image URLs from text fields (I don't know if we'll need to create some special flag for these kinds of text fields where we expect image URL inputs) and pass them into the API via the AI's approved method for passing in image URLs. 

In other words, AI will accept image urls, but not when they are passed in via a text box.

**Comment by OmarIthawi** [comment]
Hi @jswope00!

You can attach an image, which ChatGPT UI will process. URLs like you said aren't processed.

I'm not sure what's the API equivelant for that in the ChatGPT API.
